test/package/test_torchscript.py

```diff
@@ -412,13 +421,13 @@ def test_load_shared_tensors(self):
         importer = PackageImporter(buffer)
         loaded_mod_1 = importer.load_pickle("res", "mod1.pkl")
 
-        self.assertTrue(
```

Comment:
lol how did these tests ever work

Reply:
Haha yeah, I guess `_cdata` was getting cast to a bool, and since it's always going to be nonzero, it's True